CHANGELOG.md

```diff
@@ -4,12 +4,58 @@
 
 ### API changes
 
+#### Added
+
+- added `dc_context_t*   dc_context_new_closed(const char* dbfile)`, `int  dc_context_open(dc_context_t *context, const char* passphrase)`, `int  dc_context_is_open(dc_context_t *context)` and `uint32_t  dc_accounts_add_closed_account(dc_accounts_t* accounts)` #2956
```

Comment:
```suggestion
- added `dc_context_new_closed()`, `dc_context_open()`, `dc_context_is_open()` and `dc_accounts_add_closed_account()` #2956
```

adding whole signature plus variable names seems to be too much, in general, we do not do that, consulting the reference is needed anyway.